### PR TITLE
[WIP] emr api log added

### PR DESCRIPTION
**Pull request by anuragtr**
emr api exception log added

### CODE REVIEW

**Comment by miteshvp** [comment]
[test]

**Comment by lucky-suman** [comment]
[test]

**Comment by yzainee** [comment]
[test]

File: f8a_report/report_helper.py [hunk 1]
```diff
@@ -255,8 +255,8 @@ def store_training_data(self, result):
                     # This EMR invocation happens for all ecosystems almost at the same time.
                     # TODO - find an alternative if there is a need
                     self.invoke_emr_api(bucket_name, eco, model_version, github_repo)
-                except Exception:
-                    continue
+                except Exception as e:
+                    logger.exception('Unable to invoke EMR API. Reason: %r' % e)
```
Comment (miteshvp) [inline f8a_report/report_helper.py]: can we use logger.error here?

Comment (lucky-suman) [reply f8a_report/report_helper.py]: OK, will do it.

**Pull request merged**